projects/angular-keyboard/src/lib/fake-input/fake-char/fake-char.component.scss

```diff
@@ -0,0 +1,4 @@
+.char-spacing {
+    min-width: 3px;
+    display: inline-block;
```

Comment:
I'd just call the class "char". If the display is inline block, doesn't the font look bad? For example is a "g" still aligned on the baseline? Can you post a screenshot?

Reply:
The alignment etc isn't changed, the texts looks identical to me. The only issue is that new line characters are now just treated as white space. I could not find any css solution to this. 

![Screenshot 2021-02-12 at 11 19 38](https://user-images.githubusercontent.com/17401551/107773799-77308300-6d3e-11eb-9480-b3328cad8278.png)